test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/MapTest.scala

```diff
@@ -1419,6 +1419,23 @@ trait MapTest {
       assertEquals("def", mp.get("nullable"))
     }
   }
+
+  @Test def additionToKeySet(): Unit = {
+    val chm = factory.empty[String, String]
+    val set = chm.keySet()
```

Comment:
Simply `val set = factory.empty[String, String].keySet()`? (`chm` is unused later).

Reply:
Fixed in https://github.com/scala-js/scala-js/pull/4367/commits/4c21243134bca29258c37d1618657d561e6941cd